<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="baudRateBox">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>42</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="serialPortInfoListBox">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>10</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QLabel" name="baudRateLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率</string>
   </property>
  </widget>
  <widget class="QLabel" name="serialPortLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>串口</string>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>描述：</string>
   </property>
  </widget>
  <widget class="QLabel" name="manufactureLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>制造信息：</string>
   </property>
  </widget>
  <widget class="QLabel" name="serialNumLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>串口号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="locationLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>定位信息：</string>
   </property>
  </widget>
  <widget class="QLabel" name="vidLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>厂家序列号</string>
   </property>
  </widget>
  <widget class="QLabel" name="pidLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>产品序列号</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dataBitsBox">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>73</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="parityBox">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>104</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dataBitsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>数据位</string>
   </property>
  </widget>
  <widget class="QLabel" name="parityLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>奇偶校验位</string>
   </property>
  </widget>
  <widget class="QLabel" name="stopBitLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>停止位</string>
   </property>
  </widget>
  <widget class="QLabel" name="flowControlLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>控制流</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stopBitBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="flowControlBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="localEchoCheckBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>本机响应</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
